src/Generator.Rewrite/Program.cs

```diff
@@ -301,19 +301,19 @@ private static void RemoveSupportingAttributes(TypeDefinition type)
                 generatedVariables = EmitConvenienceWrapper(wrapper, native, difference, body, il);
             }
 
-            if (slot != -1)
+            if (Options.UseDLLImport)
```

Comment:
Change in logic. Explanation? Looks to me like it should be if "slot == -1 || Options.UseDLLImport"

Reply:
Yes, you're absolutely correct.